### PR TITLE
[New Framework] Nim/Prologue

### PR DESCRIPTION
**Pull request by rishavs**
Hi. I have added a basic plaintext and json benchmarks for the new Nim framework [Prologue](https://github.com/planety/prologue). More details about it can be seen here https://planety.github.io/prologue/

I will be adding the database benchmarks later.

### CODE REVIEW

File: frameworks/Nim/prologue/prologue.dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM nimlang/nim:latest
```
Comment (nbrady-techempower) [inline frameworks/Nim/prologue/prologue.dockerfile:1]: Please tie this down to a specific version for reproducability/stability

Comment (rishavs) [reply frameworks/Nim/prologue/prologue.dockerfile:1]: I started with `nimlang/nim:1.14` but that doesn't seems to be supported.
What's the current latest nim version that TFB supports?

Comment (nbrady-techempower) [reply frameworks/Nim/prologue/prologue.dockerfile:1]: It's coming from dockerhub. We don't choose the support for these https://hub.docker.com/r/nimlang/nim/tags

File: .travis.yml [hunk 2]
```diff
@@ -63,7 +63,7 @@ env:
     - "TESTLANG=Lua"
     - "TESTLANG=Mumps"
     - "TESTLANG=Nim"
-    - 'TESTDIR="Nim/httpbeast Nim/jester Nim/basolato"'
+    - 'TESTDIR="Nim/httpbeast Nim/jester Nim/basolato Nim/prologue"'
```
Comment (nbrady-techempower) [inline .travis.yml:66]: Also, please remove this line. The line above it tests all Nim frameworks

Comment (rishavs) [reply .travis.yml:66]: It was there previously as well. But I will remove the 2nd line.

Comment (rishavs) [reply .travis.yml]: Done

**Review by msmith-techempower — COMMENTED** [review]
This is one of the greatest commit messages ever. I applaud your resolve!

**Pull request merged**